inc/network-dashboard.php

```diff
@@ -0,0 +1,166 @@
+        </div>
+    </div>
+    <style>
+        /* Grid container */
+        .hale-dashboard-grid {
+            display: grid;
+            grid-template-columns: 3fr 1fr;
+            gap: 20px;
+            margin-top: 20px;
+        }
+
+        /* Grid item */
+        .hale-dashboard-item {
+            display: flex;
+            flex-wrap: wrap;
+            background: #f9f9f9;
+            border: 1px solid #ddd;
+            border-radius: 4px;
+            padding: 20px;
+        }
+
+        .hale-dashboard-left {
+            flex: 1;
+            padding-right: 20px;
+            border-right: 1px solid #ddd;
+        }
+
+        .hale-dashboard-right {
+            flex: 2;
+            padding-left: 20px;
+        }
+
+        .hale-dashboard-left h4,
+        .hale-dashboard-right h4 {
+            margin-top: 0;
+        }
+
+        /* Green box for "On:" */
+        .status-on {
+            display: inline-block;
+            background-color: #28a745; /* Green background */
+            color: #fff; /* White text */
+            padding: 2px 8px;
+            border-radius: 4px;
+            font-weight: bold;
+        }
+
+        /* Optional: Style for "Off:" */
+        .status-off {
+            display: inline-block;
+            background-color: #dc3545; /* Red background */
+            color: #fff; /* White text */
+            padding: 2px 8px;
+            border-radius: 4px;
+            font-weight: bold;
+        }
+    </style>
+    <?php
+}
+
+/**
+ * Function to retrieve the value of an environment variable.
+ * Checks both ENV and SERVER
+ * 
+ * @param string $key The name of the environment variable.
+ * @param mixed $default A default value if the environment variable is not set.
+ * 
+ * @return string The value of the environment variable or the default value.
+ */
+function hc_get_env_variable($key, $default = '') {
+    // Check if the environment variable is set in $_ENV or $_SERVER
+    $value = getenv($key) ?: ($_ENV[$key] ?? $_SERVER[$key] ?? $default);
+
+    return $value;
+}
+
+/**
+ * Detect if the WB_CONFIG cookie is present.
+ *
+ * @return bool True if the cookie is present, false otherwise.
+ */
+function hc_is_waf_bypass_cookie_present() {
+    // Check if the WB_CONFIG cookie is set and not empty
+    if (isset($_COOKIE['WB_CONFIG']) && !empty($_COOKIE['WB_CONFIG'])) {
+        return true;
+    }
+
+    return false;
+}
+
+add_action('init', 'hc_set_waf_bypass_cookie');
+
+/**
+ * Set a WAF bypass cookie for logged-in users if the setting is enabled.
+ * // Excludes subscribers (they will still get WAF)
+ */
+function hc_set_waf_bypass_cookie()
+{
+    // Value provided into the container via GitAction secrets
+    $wb_config_env_value = hc_get_env_variable('WB_CONFIG');
+
+    // Setting to edit posts excludes subscribers
+    if (current_user_can('edit_posts')) {
+        setcookie('WB_CONFIG', $wb_config_env_value, time() + DAY_IN_SECONDS, COOKIEPATH, COOKIE_DOMAIN);
```

Comment:
Does cookie_domain use the current site as otherwise it wont work with custom domains

Reply:
This is set to false...so ti should change per site

Reply:
We will test this